seed/seed.json

```diff
@@ -167,7 +173,7 @@
             "name": "BraveAds.DoNotShowCustomAdNotificationsOnNightlyChannelStudy",
```

Comment:
same as above, as this is the default this study seems to do nothing

Reply:
These studies have `max_version` and are for backwards compatibility